Hyper/Controllers/Bots/Spawn/SphericalBotSpawnStrategy.cs

```diff
@@ -37,7 +37,18 @@ public override void Spawn()
 
     public override void Despawn()
```

Comment:
Do we maybe want to spawn bots if there are fewer than 7 in spherical?

Reply:
Spawn method will generate a new bot once we kill one, so I don't think this is necessary